tox.ini

```diff
@@ -63,9 +64,16 @@ basepython =
 basepython=python
 extras=linter
 commands=
-  flake8 {toxinidir}/web3 {toxinidir}/ens {toxinidir}/ethpm {toxinidir}/tests --exclude {toxinidir}/ethpm/ethpm-spec
-  isort --recursive --check-only --diff {toxinidir}/web3/ {toxinidir}/ens/ {toxinidir}/ethpm/ {toxinidir}/tests/
-  mypy -p web3 -p ethpm -p ens --config-file {toxinidir}/mypy.ini
+    flake8 {toxinidir}/web3 {toxinidir}/ens {toxinidir}/ethpm {toxinidir}/tests --exclude {toxinidir}/ethpm/ethpm-spec
+    isort --recursive --check-only --diff {toxinidir}/web3/ {toxinidir}/ens/ {toxinidir}/ethpm/ {toxinidir}/tests/
+    mypy -p web3 -p ethpm -p ens --config-file {toxinidir}/mypy.ini
+
+[testenv:benchmark]
+basepython=python
+commands=
+    python {toxinidir}/web3/tools/benchmark/main.py --num-calls 5
+    python {toxinidir}/web3/tools/benchmark/main.py --num-calls 50
```

Comment:
Just curious - what's the reasoning behind testing 5, 50, and 100 calls here and not just checking one?

Reply:
Once the suite is more mature, we should have vectors for benchmarking (the average of) 1 call vs. 1 call; this would be useful for comparing the HTTPProvider vs. IPCProvider, for example. I think making many calls is the best way to  showcase the difference between AsyncHTTPProvider vs. HTTPProvider performance.